homeassistant/components/unifiprotect/sensor.py

```diff
@@ -399,6 +401,59 @@ def _get_alarm_sound(obj: Sensor) -> str:
 )
 
 
+CAMERA_TRIP_SENSORS: tuple[ProtectSensorEntityDescription, ...] = (
```

Comment:
Could we just make these "normal" sensors and completely get rid of the `ufp_last_trip_value` value? i.e. add them to `CAMERA_SENSORS`, `SENSE_SENSORS`, etc?

Reply:
I tried that first but it was less code to make another entity class as otherwise we needed to implement a separate `get_ufp_value` function for each one

Reply:
You should just be able to replace `ufp_last_trip_value` with `ufp_value`. It looks like the trip sensor class is just using `get_nested_value` which is all `get_ufp_value` does.

https://github.com/home-assistant/core/blob/dev/homeassistant/components/unifiprotect/models.py#L30

If you need to type cast or something, probably could do some nonsense with `functools.partial`.

Reply:
80d5e6fe639cf4ef770b595e1e85e496fcfbca6d